components/logistics-landing.tsx

```diff
@@ -1,14 +1,14 @@
 "use client";
 
-import { Truck, MapPin, BarChart, Shield } from "lucide-react";
```

Comment:
@timursaurus this page is not needed yet, according to Stanislav's requirements. That's why I removed this page. What I mean is that it’s not worth doing it yet. 

We may restore it in the future, so the changes will still apply